Classes/Service/FileGenerator.php

```diff
@@ -1151,10 +1151,9 @@ protected function generateLocallangFileContent(string $fileNameSuffix = '', str
             if ($variableName === 'domainObject') {
                 $filenameToLookFor .= '_' . $variable->getDatabaseTableName();
             }
-            $existingFile = $filenameToLookFor . '.xlf';
 
-            if (@file_exists($existingFile)) {
-                $existingLabels = $this->localizationService->getLabelArrayFromFile($existingFile, 'default');
+            if (@file_exists($filenameToLookFor)) {
```

Comment:
Just from looking, does the file have the .xlf ending? You removed line 1154, that's why.

Reply:
That's because the file ending is added twice. So the file is f.e. `locallang.xlf.xlf` when the code checks, if this file exists.
In line 1137 the filename suffix is added for the first time.
<img width="1107" alt="Bildschirmfoto 2022-09-21 um 08 31 50" src="https://user-images.githubusercontent.com/107543296/191432192-98b77f3d-8c96-4d90-ae5a-66ff651d700e.png">

<img width="329" alt="Bildschirmfoto 2022-09-21 um 08 31 58" src="https://user-images.githubusercontent.com/107543296/191432230-d1138895-20cd-4627-9cee-3cd4828a3120.png">


But i now see, that the error is still happening at other positions.

<img width="994" alt="Bildschirmfoto 2022-09-21 um 08 34 08" src="https://user-images.githubusercontent.com/107543296/191432254-3458225b-1fc0-4ce3-8a00-04e582171dea.png">